api/lib/models/BaseObject.ts

```diff
@@ -25,6 +25,13 @@ export default abstract class BaseObject {
       },      {});
   }
 
+  public get cleanRepresentation() {
```

Comment:
This function could use a doc string about what it's meant for, to me its a little confusing

Reply:
I will add it

Reply:
added